### ISSUE TITLE
Physical Entity: Core fields for physical entitites that will not be catalogued as part of main collection.

### ISSUE THREAD

**Issue by rintoult** [AAFC-BICoE/dina-planning#169]
GIVEN I have accessed DINA as collection manager

WHEN i have some material I want to inventory but not accession/catalogue in the public collection

THEN there should be a location to store data in DINA

AND I should be able to search it and catalogue storage locations of said material and potentially derive other physical entities or catalogued objects from these entities.

**Comment by rintoult** [comment]
I have no input on this one yet - hopefully I wrote the bug you wanted @dshorthouse but I still don't see how i might use this variety of entity in my collection. I need more examples of what these could be and which other modules they would relate to.

**Comment by ssbilkhu** [comment]
Would this just be a sub collection within DINA?

**Comment by jmacklin** [comment]
Yes, this makes good sense to me. We also want researchers to use DINA to catalogue their research collections and I think they would best be placed as sub-collections.

**Comment by dshorthouse** [comment]
Likewise, you can create a collection without associating it with an institution - a floater. We do not yet have pipelines for publishing data to GBIF, but this would be one logical, high-level check despite a material sample marked as publicly-releasable. If there's no declaration of institutional ownership because the collection is not explicitly linked to the AAFC institution, the doors are closed.
